src/Hl7.Fhir.Patch/PatchDocument.cs

```diff
@@ -0,0 +1,128 @@
+/* 
+ * Copyright (c) 2020, Firely (info@fire.ly) and contributors
+ * See the file CONTRIBUTORS for details.
+ * 
+ * This file is licensed under the BSD 3-Clause license
+ * available at https://github.com/FirelyTeam/fhir-net-api/blob/master/LICENSE
+ */
+
+using System;
+using System.Collections.Generic;
+using Hl7.Fhir.ElementModel;
+using Hl7.Fhir.Patch.Adapters;
+using Hl7.Fhir.Patch.Exceptions;
+using Hl7.Fhir.Patch.Internal;
+using Hl7.Fhir.Patch.Operations;
+using Hl7.Fhir.Specification;
+
+namespace Hl7.Fhir.Patch
+{
+    // Implementation details: the purpose of this type of patch document is to allow creation of such
+    // documents for cases where there's no class/DTO to work on. Typical use case: backend not built in
+    // .NET or architecture doesn't contain a shared DTO layer.
+    public class PatchDocument : IPatchDocument
+    {
+        public List<Operation> Operations { get; }
```

Comment:
Perhaps making there fields private? Maintaining this list is done by the method `Add`